src/lib389/lib389/__init__.py

```diff
@@ -387,6 +387,9 @@ def __init__(self, verbose=False, external_log=None):
         self.uuid = str(uuid.uuid4())
         self.verbose = verbose
         self.serverid = None
+        # Are we a container? We get containerised in setup.py during SetupDs, and
+        # in other cases we use the marker to tell us.
+        self._containerised = os.path.exists(DSRC_CONTAINER) or containerised
```

Comment:
IMHO we should also be in that case if getuid() is not 0 (for prefixed install running as plain user)
   ( I hit the issue on Friday (and FYI, I also got in trouble with ldapi sockets that could not be created)

Reply:
No - I think this isn't a correct way to determine the containerisation state of the python tools. A user might do "sudo -u dirsrv dsconf ..." for ldapi, and this would confuse that on a local install.

Reply:
I am not sure that "sudo -u dirsrv dsconf" works nicely anyway (well dsconf may work but dsctl does not (semanage calls will fail)  but if you would rather check for the prefix variable I have not problem with that.

Reply:
It does work pretty well dsconf at least :) dsctl is a bit of a weirder one.